src/server/lib/__tests__/rate-limit/rateLimitMiddleware.test.ts

```diff
@@ -242,9 +254,43 @@ describe('rate limiter middleware', () => {
     }
   });
 
+  it('should not apply the rate limiter when logOnly is set to true', async () => {
+    const rateLimiterConfig: RateLimiterConfiguration = {
+      enabled: true,
+      settings: {
+        logOnly: true,
+        trackBucketCapacity: false,
+      },
+      defaultBuckets: {
+        enabled: true,
+        globalBucket: { capacity: 1, addTokenMs: 500 },
+      },
+    };
+
+    // Start the application server.
+    const { server, teardownServer } = await getServerInstance(
+      rateLimiterConfig,
+    );
+
+    try {
+      // Confirm that we are not rate limited when logOnly is set to true.
+      await request(server).get('/signin').expect(200);
+      await request(server).get('/signin').expect(200);
+
+      await teardownServer();
+    } catch (error) {
+      await teardownServer();
+      throw error;
+    }
```

Comment:
Could use a `finally` block for the teardown.

Reply:
great idea, I'll action this 🔨